src/obstacle_avoidance/test/my_node_rostest.cpp

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
We're not doing an ROSTests for this node, so this file can probably be removed.